src/modules/render.js

```diff
@@ -86,6 +86,13 @@ const RENDER = (meals) => {
       </div>
     `;
       getComments(meal.id).then((d) => {
+        const numofcom = document.getElementById('numofcom');
```

Comment:
I think it would be better if you name your variables in a way that they reveal the intention behind it. This way they become searchable and easier to understand after a person sees it. 😊 You can learn more about [clean code best practices in this link](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices)